src/DatadirTestsFromDirectoryProvider.php

```diff
@@ -75,6 +76,13 @@ protected function processOneTest(SplFileInfo $testSuite): void
             $expectedOutputDirectory = $outTemplateDir;
         }
 
+        if (file_exists($expectedReturnCodeFile)) {
+            $returnCode = trim(file_get_contents($expectedReturnCodeFile));
+            if (preg_match('/^[012]$/', $returnCode)) {
+                $expectedReturnCode = (int) $returnCode;
+            }
```

Comment:
Nevim, jestli bych neudělal else a v něm výjimku - pokud se upíšu (tj. dám tam třeba 3) tak se to bude chovat tak, že to čeká > 0 - takže to projde a nijak mě to neupozorní, že tam mám neplatnou hodnotu.